### ISSUE TITLE
Expose CANTalon.setPosition(double) in TalonSRX

### ISSUE THREAD

**Issue by agausmann** [strongback/strongback-java#68]
It is currently impossible to truly zero the TalonSRX's encoder input if you are using the closed-loop control. Calling `talon.getEncoderInput().zero()` simply zeroes the angle as seen by the user, but not as seen by the Talon's processor.

**Comment by rhauch** [comment]
It's not possible because it is unnecessary.

When you work with a Talon SRX device in close-loop mode, you use a `TalonController` (which extends both `TalonSRX` and `PIDController`) and configure it such that the encoder device wired into the Talon SRX device is properly represented with a `Gyroscope`. Yes, the `Gyroscope` object can be zeroed, but when you call `zero()` it simply remembers the reading at that point and subtract it from all subsequent angles returned by the `Gyroscope`.

Then, to command the Talon SRX device to move the motor until the encoder reaches a designated angle using closed loop control, your code should call `TalonController.withTarget(value)` and supply the desired angle as would be read by the `Gyroscope` (which presumably you've previously zeroed). The implementation of this method is [here](https://github.com/strongback/strongback-java/blob/master/strongback/src/org/strongback/hardware/HardwareTalonSRX.java#L96), and it adds the zero point back in _then_ instructs the Talon SRX's close loop control. (Note that the `zero` value recorded in the internal `EncoderInputSensor` class is a measure of the raw output of the encoder, not the actual angle.)

The bottom line is that the _zero_ behavior is removed from the `Gyroscope` output _and_ added back in when setting the target for the close-loop control.

Does this make sense? Or did I misunderstand your statement?

**Comment by agausmann** [comment]
If I understand what you are saying, the `TalonController.withTarget(target)` uses the offset that is set when `getEncoderInput().zero()` is called?

**Comment by rhauch** [comment]
@AGausmann, yes. If you look at [the code](https://github.com/strongback/strongback-java/blob/master/strongback/src/org/strongback/hardware/HardwareTalonSRX.java#L96), you'll see that the zero value is added back in to the raw target value for the encoder before it is pushed down to the Talon SRX device.

**Comment by agausmann** [comment]
Okay, thanks.
